src/main/java/org/wso2/carbon/connector/Append.java

```diff
@@ -45,7 +45,11 @@ public void connect(MessageContext messageContext) throws ConnectException {
             }
         } catch (Exception e) {
             handleException("Error while connecting the server or calling the redis method", e, messageContext);
+        } finally {
```

Comment:
jedis is not within the scope. Need to move it about try block before checking whether it is null.  And also remove the jedis.disconnect() also. Thanks for feedback. We will incoroporate all these changes and rerelease.